src/client/TerritoryPatterns.ts

```
@@ -0,0 +1,88 @@
+  }
+
+  static getSelectedPatternBase64(): string | undefined {
+    return (
+      localStorage.getItem(TerritoryPatternStorage.BASE64_KEY) ?? undefined
+    );
+  }
+
+  static setSelectedPatternBase64(base64: string): void {
+    localStorage.setItem(TerritoryPatternStorage.BASE64_KEY, base64);
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)